### DISCUSSION TITLE
[Question]: do save chat content

### DISCUSSION THREAD

**Discussion by lichq5** [danny-avila/LibreChat#2252] [Q&A]
### What is your question?

 if I don't want to store chat content into database, is there any setting could get it done, or I have to change the code?
 cause LibreChat runs in docker, so how do I deploy may version into the docker if it have to change the code.

### More Details

no more details

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
The app is designed to be stateful. if you want transient chats, you should manually clear them as needed

![image](https://github.com/danny-avila/LibreChat/assets/110412045/f2bcd44c-ecec-48d4-8046-c2664db42805)

**Reply by danny-avila** [reply]
It also doesn't "need" to run on docker. You can easily run it locally: https://docs.librechat.ai/install/index.html

**Reply by lichq5** [reply]
thanks for your reply. but what I want to do is not saving any contents in db during chatting instead of clear chats manully after chats already write down to mongo db. so for doing that, are I have to modify some code or disable some sort of features can handle this?

**Comment by lichq5** [comment]
if I change libreChat's code, how to deploy it as a docker image?

**Reply by fuegovic** [reply]
if you change the code and still want to use it in docker you'll have to build it instead of using the image, see: https://docs.librechat.ai/install/configuration/docker_override.html#examples

(just after "if you want to locally build the image for the api service...")

**Reply by lichq5** [reply]
ok, but after following the guidline of "if you want to locally build the image for the api service", it just could not start docker normally:
<img width="827" alt="image" src="https://github.com/danny-avila/LibreChat/assets/13826559/a468755e-8866-46e7-9385-3943a76f4b6f">
